Change qbittorrent logo. Issue #6467.

sledgehammer999 opened the pull request:
All pngs where created via GIMP. The .ico file was created by taking the relevant pngs and using `icotool` program. The .icns file was created by taking the relevant pngs and using `png2icns` program.  
I wasn't able to find a program for the OS/2 .ico format (different from the Windows .ico) until I started commiting and compiling. But after that I found 2 promising programs [Deark](http://entropymine.com/deark/) and [Image Alchemy](http://fileformats.archiveteam.org/wiki/Image_Alchemy). I will try them later.

Caveats:
1. `src/icons/skin/splash.png` has the old log embedded
2. `src/qbittorrent_file.ico` wasn't recreated. I think I can do it if use the new logo plus `src/icons/qbt-theme/text-plain.png` from the old theme. Same goes for `dist/mac/qBitTorrentDocument.icns`

Pics:
![file_icon](https://cloud.githubusercontent.com/assets/273315/23664839/679e21f6-035f-11e7-8786-ccfb7413ca7f.jpg)
![program](https://cloud.githubusercontent.com/assets/273315/23664848/6b7f7d06-035f-11e7-81f0-d0a34b65a69a.jpg)

Notice how crappy the icon looks as an window-icon and as tray-icon? I think this is actually a regression of qt with 5.7.0 (or 5.7.1 don't remember). Can you confirm that it looks crap on yours too?

LordNyriox commented:
@sledgehammer999:  In GIMP I personally prefer to use SVG source files, then load them scaled to several different sizes in GIMP.  I have an entire workflow for generating Windows .ico files, and I know how to generate complete mac .icns files.  

If I create a set of Windows and Mac program icons, would you use them (they will have a fairly large filesize, but they will be top quality).

sledgehammer999 commented:
@LordNyriox Just so you know, I inspected the old icns and ico files to see what sizes they used. And then I used same sizes for the new icon. I have no problem using files created by you.
PS: Does that mean that you will recreate the various pngs too? If yes, can you do the same for the 2nd logo posted [here](https://github.com/qbittorrent/qBittorrent/issues/6467#issuecomment-284742493) too?

LordNyriox commented:
@sledgehammer999:  That's the plan.  Regenerated PNGs from SVGs as well as both recolored and original new icon styles.   

Note, most built-in Windows/Mac icons used in applications/programs have fewer icon sizes than those supported, or have them included in an unsupported configuration that messes up scaling.  I've figured out .ico through trial and error, and .icns is easy with the right script and proper formatting (which I happen to know).  

I'll have them done sometime in the next two days.  I have to borrow my mother's mac for the .icns, but the .ico files should be ready shortly.  

Thanks again, 
– Nyriox

bugmenot2 commented:
Hi @LordNyriox, my design lacks vectorial content completely, so can't be exported to SVG... :blush:

bugmenot2 commented:
Hi @LordNyriox, my design lacks vectors completely, so can't be exported to SVG... :blush:

LordNyriox commented:
@bugmenot2:  Out of curiosity, how did you make your version of the icon?  

Perhaps I could re-create the process in SVG.  

Thanks for your reply.

bugmenot2 commented:
>Sorry to bug you :grin:

Please stop deleting your comments, it's annoying!

I started with 2 layers (one for the text and another for the orb), then rendered both to be able to use image-oriented tools and effects (I couldn't do what I wanted with vectors).

If you really need a SVG for the job you can embed a picture like this:
```svg
<?xml version="1.0" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
<svg id="test" width="512px" height="512px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
	<image x="0px" y="0px" width="512px" height="512px" xlink:href="whatever.png"></image>
</svg>
```

sledgehammer999 commented:
IMO, better focus on the svg one. Sorry @bugmenot2. Thanks for the contribution though.

bugmenot2 commented:
Excuse me @sledgehammer999, 1024 x 1024 for an icon isn't enough?

sledgehammer999 commented:
>Excuse me @sledgehammer999, 1024 x 1024 for an icon isn't enough?

It is enough, but if @LordNyriox prefers to work with svg, I just told him to let go of trying to get an svg version of your logo.

Generally we're trying to migrate to svg so we can load them natively in Qt without relying on generated pngs. Presumably this will help with scaling and highDPI screens.

bugmenot2 commented:
I'm fine working with @LordNyriox to get this job done, I also have the necessary knowledge/toolset to build icons and PNGs for the program :smile:

bugmenot2 commented:
# [:bangbang:**SVG**:bangbang:, Windows ICO and 14 PNGs this way :sunglasses:](https://github.com/qbittorrent/qBittorrent/files/832625/qBittorrent.zip)

bugmenot2 commented:
# [:bangbang:**SVG**:bangbang:, Windows ICO and 14 PNGs this way :sunglasses:](https://github.com/qbittorrent/qBittorrent/files/832625/qBittorrent.zip)

surfernsk commented:
@bugmenot2 Why in svg the link to png?
`<image x="0px" y="0px" width="1024px" height="1024px" xlink:href="qBittorrent-1024.png">`

bugmenot2 commented:
[OK... Here's a 100% self-contained SVG for your viewing pleasure, anything else sir?](https://github.com/qbittorrent/qBittorrent/files/832903/qBittorrent-full.zip)

surfernsk commented:
Dear @bugmenot2 , thank you for the work you have done.

PS.
You were offended by my question?

bugmenot2 commented:
Nah, your question was fine, voting down my comment was a little rough though :smile:
It's all good :+1:

LordNyriox commented:
@bugmenot2:  Thanks.  I'll get right on to generating icons.

Chocobo1 commented:
First, I do like @bugmenot2 icon.
But embedding png into svg, doesn't that still be raster graphic? If so, then png will be better (because compression).
The point of svg format is to carry vector graphic. Or I'm missing something?

surfernsk commented:
@bugmenot2 Alas, the last option is not vector. You are asked to provide a vector version of the logo, if possible.

something like that - [qBittorrent-vector(draft).zip](https://github.com/qbittorrent/qBittorrent/files/834360/qBittorrent-vector.draft.zip) (My skills in owning vector graphics are very small, the quality may be insufficient)

LordNyriox commented:
@surfernsk:  Thanks for the draft.  You just saved me an hour's worth of work (I need to configure Inkscape again).

atfzl commented:
Hi @LordNyriox,

The png that you have exported have a lot of aliasing.
If you zoom in, you can see the edges in the curves.
Here is your uploaded png for 1024 x 1024:
![qbittorrent-new-1024](https://cloud.githubusercontent.com/assets/14862437/23809620/626e52b8-05f4-11e7-907c-cf6aecad0cc3.png)

Here is the anti-aliased version:
![qbittorrent new 1024 anti aliased](https://cloud.githubusercontent.com/assets/14862437/23809629/6d0ac8e6-05f4-11e7-995b-d12212d963b7.png)

Also the color is not correct for the mac version (.icns file)

[qbittorrent-new.icns.zip](https://github.com/qbittorrent/qBittorrent/files/834927/qbittorrent-new.icns.zip)

atfzl commented:
I'll recreate @bugmenot2's version of icon tomorrow in proper vector format in Sketch app.
I'll export both icons to all the required sizes (16, 22, 24, 32, 36, 48, 64, 72, 96, 128, 192, 256, 512, 1024) png, svg, ico and icns.

LordNyriox commented:
@atfzl:  On my Windows machine, all I did was use GIMP's built in SVG importer and ICO/PNG exporters to generate the ICO/PNG files. 

The Mac icons were created on macOS El Capitan, using the native command-line tools, from the PNG files I created on my Windows machine.  

I did not perform any kind of anti-aliasing task, mostly because I'm don't have the right tools.  

If you make the PNG files, I can gladly regenerate the ICO and ICNS files from your sources.  (Both Windows and Mac are finicky about icon sizes, and I have spent years figuring out how to make the scaling behave).  

Thanks again, 
– Nyriox

LordNyriox commented:
@atfzl:  Also, the color is correct for the Mac version, but I included Retina versions (2x size) of all the icons.  Mac doesn't seem to handle pre-scaling and auto-scaling for the gradients the same way.

bugmenot2 commented:
>I have an entire workflow for generating Windows .ico files, and I know how to generate complete mac .icns files.

>If I create a set of Windows and Mac program icons, would you use them (they will have a fairly large filesize, but they will be top quality).

>On my Windows machine, all I did was use GIMP's built in SVG importer and ICO/PNG exporters to generate the ICO/PNG files.

>I did not perform any kind of anti-aliasing task, mostly because I'm don't have the right tools.

...

![Triple Facepalm](https://cloud.githubusercontent.com/assets/377017/23814793/ba62e726-05c3-11e7-8930-b8a12b986517.jpg)

bugmenot2 commented:
>I have an entire workflow for generating Windows .ico files, and I know how to generate complete mac .icns files.

>If I create a set of Windows and Mac program icons, would you use them (they will have a fairly large filesize, but they will be top quality).

>On my Windows machine, all I did was use GIMP's built in SVG importer and ICO/PNG exporters to generate the ICO/PNG files.

>I did not perform any kind of anti-aliasing task, mostly because I'm don't have the right tools.

...

![Epic Triple Facepalm](https://cloud.githubusercontent.com/assets/377017/23818811/b55687be-05dd-11e7-9225-40a0763d8bf0.jpg)

LordNyriox commented:
@bugmenot2:  Thank you for providing me with a new forum signature.  ;}

atfzl commented:
Here are the exported resources of first icon:
[qbittorrent-new-logo.zip](https://github.com/qbittorrent/qBittorrent/files/835592/qbittorrent-new-logo.zip)

Will soon update for the second icon.

atfzl commented:
Both logo with original source sketch file and exported resources(only .ico is not there): 

[qbittorrent-new.zip](https://github.com/qbittorrent/qBittorrent/files/835597/qbittorrent-new.zip)

surfernsk commented:
@LordNyriox, @atfzl, @sledgehammer999 that about [qbit.zip](https://mega.nz/#!eVcwVbBZ!Zkov2cjcqulyK08etfLQmCfTzc6UwgQKzulr8xnMBcY) [1024px](https://mega.nz/#!DQlX2ZYC!9qRxvk9b25CPSw2LigDRu1TQQjbwaTDeFw-dv4ryOdM)

LordNyriox commented:
@atfzl, @sledgehammer999, @Chocobo1:  Here's my Windows .ICO files generated from @atfzl's new icon set.  

[<https://mega.nz/#!3J8RFS6T!TKFYtTACLtcH1Nbi27sQzNk3v6GyqgtlNygngD4YUWo>] 

Thanks again for the fun debate; it's not often I find someplace to discuss icon preparation techniques.  :}

sledgehammer999 commented:
I like @atfzl new svg too. @surfernsk your mod looks good too but nowadays my preference tends to the "flat design" side. I could be wrong. I have to look at the smaller icons too. There they might look much better.

Thx for the contributions guys, I'll try to look at them this weekend.

bug-me-not commented:
‍	@sledgehammer999	@sledgehammer999 	@sledgehammer999	@sledgehammer999
‍	@sledgehammer999	:whale::whale::whale::whale::whale::whale::whale::whale::whale::whale::whale::whale::whale::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::whale::whale::whale::whale::hibiscus::whale::whale::hibiscus::whale::whale::hibiscus::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::whale::whale::whale::whale::hibiscus::whale::whale::whale::whale::whale::hibiscus::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::whale::whale::whale::whale::hibiscus::whale::whale::hibiscus::whale::whale::hibiscus::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::whale::whale::whale::whale::hibiscus::whale::whale::hibiscus::whale::whale::hibiscus::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::hibiscus::hibiscus::hibiscus::hibiscus::hibiscus::whale::whale::hibiscus::whale::whale::hibiscus::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::whale::whale::whale::whale::hibiscus::whale::whale::hibiscus::whale::whale::hibiscus::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::whale::whale::whale::whale::hibiscus::whale::whale::hibiscus::whale::whale::hibiscus::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::whale::whale::whale::whale::hibiscus::whale::whale::hibiscus::whale::whale::whale::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::hibiscus::whale::whale::whale::whale::hibiscus::whale::whale::hibiscus::whale::whale::hibiscus::whale:	@sledgehammer999
‍	@sledgehammer999	:whale::whale::whale::whale::whale::whale::whale::whale::whale::whale::whale::whale::whale::whale:	@sledgehammer999
‍	@sledgehammer999	@sledgehammer999 	@sledgehammer999	@sledgehammer999

LordNyriox commented:
@sledgehammer999 approveFeature

Inamaru commented:
@sledgehammer999 I was wondering if I still got the chance to make some logo design before this is approved.

maverick74 commented:
I would just like to mention the icon on https://github.com/qbittorrent/qBittorrent/issues/5114 that is also very cool (not very "traditional", but cool)

adxanim commented:
how about these...
![qb4](https://user-images.githubusercontent.com/30808176/29093638-22d917c0-7ca8-11e7-9814-c80ca224cb7f.png)
![qb5](https://user-images.githubusercontent.com/30808176/29093639-24cec408-7ca8-11e7-8ff9-90343e3113b3.png)

kaasplank commented:
I just found out this github page exists but here's my idea. :)

![artboard 1](https://user-images.githubusercontent.com/31142286/30395967-4d9d40e4-98c8-11e7-9981-2ffd0b88e849.png)

sledgehammer999 commented:
Hey guys, I am back again to this again.
It is my understanding that most of you prefer @bugmenot2 (or @bug-me-not) proposal and banded together to edit it up to par? And i think the last accepted edit was by @atfzl, which he would supposedly recreated as true SVG? Is there a link to it?
Am I wrong on my assumptions?

If I am not wrong, please point me to the appropriate comment. And to possible comments that posted PNG/ICO/ICNS exports.

@maverick74 @adxanim Thanks for your proposals and effort, but frankly I don't like them.
@kaasplank Thanks for proposal and effort, I think yours is close to the ones already considered from @bugmenot2 (or @bug-me-not) and @atfzl. I like those 2 better.

LordNyriox commented:
@sledgehammer999:  

The icons from @atfzl were linked here:  [<https://github.com/qbittorrent/qBittorrent/pull/6484#issuecomment-285851287>].  

Likewise, the .ICO files I generated from the @atfzl icons were linked here:  [<https://github.com/qbittorrent/qBittorrent/pull/6484#issuecomment-285896593>].  

Thanks again!

LordNyriox commented:
@sledgehammer999:  
Also, do not forget to load the new tray icons in SVG—like the way the monochrome tray icons have been loaded since [<https://github.com/qbittorrent/qBittorrent/pull/7065>].

sledgehammer999 commented:
@bugmenot2 (or @bug-me-not) and @atfzl I assume you contribute this art under the GPLv2+? Or do you want to use a Creative Commons (or other) license?

atfzl commented:
@sledgehammer999 i am okay with GPL

maverick74 commented:
@sledgehammer999 No need to thank me!!! I didn't do anything!!! Just pointed to another proposal from another bug report i liked!!!

I guessed you would good with the one you choose - it's more look-alike the actual one, but since i also liked the other proposal i suggested it. 

I'm ok with the one chosen, anyway ;)

great job everyone

bug-me-not commented:
>Hey guys, I am back again to this.
It is my understanding that most of you prefer @bugmenot2 (or @bug-me-not) proposal and banded together to edit it up to par? And I think the last accepted edit was by @atfzl, which he would supposedly recreate as true SVG? Is there a link to it?
Am I wrong on my assumptions?
If I am not wrong, please point me to the appropriate comment. And to possible comments that posted PNG/ICO/ICNS exports.

Correction, the vectorized version of my icon set was already done and posted by @surfernsk ([link to comment](https://github.com/qbittorrent/qBittorrent/pull/6484#issuecomment-285851290))

>@bugmenot2 (or @bug-me-not) and @atfzl I assume you contribute this art under the GPLv2+?

Correct :)

sledgehammer999 commented:
>Correction, the vectorized version of my icon set was already done and posted by @surfernsk (link to comment)

@surfernsk's comment comes after @atfzl's ([link](https://github.com/qbittorrent/qBittorrent/pull/6484#issuecomment-285851287)) . And @surfernsk seems to have edited @atfzl's to add text shadow.

bug-me-not commented:
>@surfernsk's comment comes after @atfzl's (link) . And @surfernsk seems to have edited @atfzl's to add text shadow.

Not quite, if you check carefully you'll notice [**@surfernsk** did the first SVG version of my icon](https://github.com/qbittorrent/qBittorrent/pull/6484#issuecomment-285702718) *before* [**@atfzl** announced he was going to recreate my design](https://github.com/qbittorrent/qBittorrent/pull/6484#issuecomment-285762889) :)

sledgehammer999 commented:
I pushed a new commit based on the light variant of @atfzl vector. It looked a tiny bit nicer than the other two. However, after compiling and taking screenshots I noticed that at the small sizes the icons looked even smaller because the white halo/ring is almost invisible. Take a look at the following screenshots at the titlebar, taskbar, fileicon. It seems a little bit smaller that it actually is. @atfzl as you are a designer, is there any trick you can do to make it pop? Or should I try with your normal variant of the vector? Is it the same and just the ring color changes?

Screenshots:
![2](https://user-images.githubusercontent.com/273315/30723807-51de984e-9f42-11e7-907d-0ed35468e495.jpg)
![1](https://user-images.githubusercontent.com/273315/30723839-8ba6198a-9f42-11e7-95a9-a9eb6d1fdfae.jpg)

bug-me-not commented:
>@atfzl as you are a designer, is there any trick you can do to make it pop?

I'm offended. Since I know you aren't a native English speaker I'll assume you aren't suggesting that I'm not a designer or otherwise excluding us from the icon polishing process.

:bangbang: [qBittorrent.svg.zip](https://github.com/qbittorrent/qBittorrent/files/1323848/qBittorrent.svg.zip)

sledgehammer999 commented:
>I'm offended. Since I know you aren't a native English speaker I'll assume you aren't suggesting that I'm not a designer or otherwise excluding us from the icon polishing process.

I am sorry. No, I didn't mean to offend you. I refered to @atfzl because he created the svg that I used. I compared him to me(I am no designer), and requested his help.

I'll see your svg in a few hours. Thank you.

bug-me-not commented:
You're welcome @sledgehammer999 :smiley:
Don't worry, it's all good :+1:

sledgehammer999 commented:
I just tested it. Unfortunately it has the same problem at those small sizes.
Next step for me is to try out the "normal" version of @atfzl svg. I think the only difference between that and the "light" one is the color of the outer ring.

LordNyriox commented:
@sledgehammer999:  While you are working on these icons, could you also change the tray icon to use the SVG file directly?  

This was already implemented for the monochrome tray icons in [<https://github.com/qbittorrent/qBittorrent/pull/7065>].  All you need to do is extend that method to the new colored icon as well.  

Thanks.

sledgehammer999 commented:
I pushed some new commits. I think it looks ok now. Look at the screenshots at the bottom.
1. I used [svgo]() on the svg to optimize/shrink it in filesize. I also used [PNGGauntlet]() on the pngs to optimize/shrink them in filesize. @atfzl did you create the icns file from the pngs? If so, care to see if the new icons create a smaller one? click [here](https://github.com/qbittorrent/qBittorrent/files/1331314/qbittorrent-new-logo-optimized.zip)  for zip
2. The svg for the tray isn't loaded correctly by Qt. Specifically it doesn't render the letters. However, @bug-me-not / @surfernsk svg has its letters rendered by Qt. I think it has to do with the way those letters are encoded. @atfzl svg references the font, but @surfernsk svg actually has the specific glyphs (?) defined inside the svg. I don't know if I can find a solution for Qt. Could you @atfzl make the svg have the glyphs inside it too? (or someone else).

![1](https://user-images.githubusercontent.com/273315/30836721-a182c0ae-a268-11e7-86cc-7730cc8c50a8.png)
![2](https://user-images.githubusercontent.com/273315/30836724-a53a5202-a268-11e7-9c12-6bc098c68ca6.png)

surfernsk commented:
@sledgehammer999 "qb" letters is curve object. 
try this [qbittorrent-new-logo-optimized.zip](https://github.com/qbittorrent/qBittorrent/files/1332762/qbittorrent-new-logo-optimized.zip)

atfzl commented:
[qbittorrent-logo-without-font.zip](https://github.com/qbittorrent/qBittorrent/files/1332806/qbittorrent-logo-without-font.zip)

Hi, I have changed the svg to have shapes instead of referencing the font.

atfzl commented:
@sledgehammer999 

Here is the new icns file created from the png provided by you.

[qbittorrent-new-1024.png.icns.zip](https://github.com/qbittorrent/qBittorrent/files/1332847/qbittorrent-new-1024.png.icns.zip)

bug-me-not commented:
>I just tested it. Unfortunately it has the same problem at those small sizes.

OK, since SVG renderers don't seem to preserve those kind of details the same way an image scaling algorithm would I finished setting up a new SVG toolchain to make sure the icon looks right at any given size, and here are the results:

![:)](https://user-images.githubusercontent.com/24734192/30867095-5f9d10ce-a2b1-11e7-9909-b335f6c6d410.png)

These icons were rendered straight from the SVG. [Here's the edited SVG with PNGs and the assembled Windows icon.](https://github.com/qbittorrent/qBittorrent/files/1333648/qBittorrent.zip)

sledgehammer999 commented:
Finally I figured out why it looked so bad at the small sizes.
All the svgs linked here have padding.
For example @atfzl light svg has these properties at the circle element 
```
cx="512" cy="512" r="460"
```

I changed them to
```
cx="512" cy="512" r="496"
```

and then regenerated the 16 and 32 pngs. They looked great. Compared them to the old ones and I confirmed that the old ones looked bad.

So I can happily go back to my first choice. @atfzl can you convert your light version to not use fonts? Will you be able to remove the padding too? If yes, then I suggest you also proportionally increase the font size and adjust the position of the letters.
I can produce the required pngs via [svgexport](https://github.com/shakiba/svgexport). But if you want, you can produce them yourself. I don't mind. But don't produce the icns file yet. I'll still need to optimize the pngs filesize using PNGGauntlet.

bug-me-not commented:
>All the svgs linked here have padding.

[The SVG I linked above](https://github.com/qbittorrent/qBittorrent/pull/6484#issuecomment-332241713) has a ~490.6 radius, which is very close to your suggestion...

sledgehammer999 commented:
>The SVG I linked above has a ~490.6 radius, which is very close to your suggestion...

I don't know how you measured that. The 2 circle elements have `r="442.5"` and `r="475"` And launching it in firefox shows quite a lot of padding from the edges.

bug-me-not commented:
>I don't know how you measured that.

`r="475"` plus `stroke-width:15.58874989` equals 490.58874989 :)

sledgehammer999 commented:
>`r="475"` plus `stroke-width:15.58874989` equals 490.58874989 :)

Actually it is `r+(stroke-width/2)`.

bug-me-not commented:
>Actually it is `r+(stroke-width/2)`.

Yup, you're right. I'll adjust my design accordingly, thanks for the heads up :)

sledgehammer999 commented:
>Yup, you're right. I'll adjust my design accordingly, thanks for the heads up :)

No need to update your svg. I like @atfzl light version and I want to use that one.

surfernsk commented:
@sledgehammer999 atfzl lite, no font 
[atfzl_qbittorrent-new-light_no_font-optimaze.svg](https://github.com/qbittorrent/qBittorrent/files/1335843/atfzl_qbittorrent-new-light_no_font-optimaze.zip)

sledgehammer999 commented:
@surfernsk the glyphs are different. You probably don't have verdana installed. Also there is still the padding.
@atfzl any news?

surfernsk commented:
@sledgehammer999 Oops, I missed the right font.
[![qbittorrent2](https://user-images.githubusercontent.com/2264893/30952337-a471303a-a451-11e7-8a21-be83d45d922c.png)](https://drive.google.com/open?id=0B0n6sYAxOLO1dHAtZXBFb0Roc0k)

[![qbittorrent3](https://user-images.githubusercontent.com/2264893/30952658-fac66918-a452-11e7-85de-8b72b5a4dbe9.png)](https://drive.google.com/open?id=0B0n6sYAxOLO1Z18wazBDWmVBMVE)

sledgehammer999 commented:
A had a sudden question/realization: I started wondering about the Verdana license. I didn't find a definitive answer. Maybe it is a good idea to switch to "open sans" font to be sure?

surfernsk commented:
@sledgehammer999 Logo with Open Sans (optimized, 1004px) [download](https://drive.google.com/drive/folders/0B0n6sYAxOLO1Uk0tdGZhN1gzR1E)

surfernsk commented:
@sledgehammer999 What about like
![qbittorrent-light_open_sans_moded svg](https://user-images.githubusercontent.com/2264893/31134239-bd18c54c-a88b-11e7-9b19-bb4e477ff816.png)
P.S.
I was a little transformed font Open Sans

sledgehammer999 commented:
@surfernsk I proceeded on my own because this back-n-forth action takes time.

I removed the padding around the circle as mentioned earlier. Then I used the same ratio to increase the font size. I used Inkscape to re-center vertically+horizontally the bigger text. And then I converted the letters to paths. (I used the open sans font).

I have pushed new commits using the new svg with open sans font. I think this is my final decision. I have included both svgs, one that uses the font and one that uses path for the letters. For easier modification down the road. Minor tweaks can be performed afterwards with new PRs.
So, unless there is a good argument against it I am going to merge this at some point in the next days.

Here are 2 screenshots on how it looks on Windows 7:
![untitled1](https://user-images.githubusercontent.com/273315/31260279-6c759644-aa55-11e7-94c0-23b8ac6f9838.png)
![untitled](https://user-images.githubusercontent.com/273315/31260281-70e17a5e-aa55-11e7-93d7-c565b1fe64bd.png)

bug-me-not commented:
>So, unless there is a good argument against it I am going to merge this at some point in the next days.

https://qbforums.shiki.hu/index.php?action=pm

sledgehammer999 commented:
1. Commit with splash screen modification has been pushed (yeah I know, it's not very creative)
2. Commit changing the fileicon has been pushed too:
![fileicon256](https://user-images.githubusercontent.com/273315/31522990-8569cbe8-afb9-11e7-94fe-60d0c25b8d08.png)

I think I'll find time this weekend to make macOS icns files too. Then this will be merged and v4.0.0 will be released.

@bug-me-not did you see my reply on your PM?

bug-me-not commented:
>@bug-me-not did you see my reply on your PM?

Yup, please check my reply: https://qbforums.shiki.hu/index.php?action=pm

sledgehammer999 commented:
@bug-me-not One final PM in the forum by me.

bug-me-not commented:
>@bug-me-not One final PM in the forum by me.

:+1:

sledgehammer999 commented:
Merging this since I am beginning the release of v4.0.0.

Thx to all contributors, commentators and critics.

Cormak commented:
Hey guys, the new icon is not production ready on 4.0.0. It's aliased on every small size. Definitely needs work.

sledgehammer999 commented:
>Hey guys, the new icon is not production ready on 4.0.0. It's aliased on every small size. Definitely needs work.

See https://github.com/qbittorrent/qBittorrent/issues/7725#issuecomment-345696356

Chocobo1 commented on src/icons.qrc:
```diff
@@ -251,7 +251,6 @@
         <file>icons/flags/zw.svg</file>
         <file>icons/L.gif</file>
         <file>icons/loading.png</file>
-        <file>icons/qbittorrent.png</file>
```
The webUI login screen is missing the qbt logo, I guess is due to this line.

@glassez 
Since you have #7610, maybe you can fix this in your PR, or let me know to open a PR for this.

glassez commented:
Don't wait until #7610. Just return it back.

Chocobo1 commented:
>Don't wait until #7610. Just return it back.

OK, see PR #7764.
